Symfony.gitignore

```diff
@@ -39,3 +42,5 @@
 
 # Backup entities generated with doctrine:generate:entities command
 **/Entity/*~
+/.idea/
```

Comment:
This is a JetBrains rule and more nuanced than ignoring this whole folder - see [`Global\JetBrains.gitignore`](https://github.com/github/gitignore/blob/master/Global/JetBrains.gitignore)